dspback/schemas/hydroshare/schema.json

```diff
@@ -173,8 +176,8 @@
           "type": "string"
         },
         "description": {
-          "title": "HydroShare Profile",
-          "description": "A string containing the path to the hydroshare profile",
+          "title": "HydroShare Profile URL",
```

Comment:
This is not a url.  The description property is pretty weird, it is basically a combination of a user id and the url to the profile on hydroshare.